alpine/resin

```diff
@@ -0,0 +1,66 @@
+#!/sbin/openrc-run
```

Comment:
@petrosagg, I've tried the start/stop templates like that but can't find the detail docs for it, I don't know how to pass the environment variables or set the working directory for the daemon using that template. that's why I used start-stop-daemon.